lib/localization/locale_model.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+class LocaleModel extends ChangeNotifier {
+  Locale _locale = Locale('en');
```

Comment:
nieładnie w ten sposób ustawiać default value, lepiej by było inicjować w konstruktorze dając jako parametr chociazby lang code, ale jak się nie da to zostaw

Reply:
ok zmienię